### ISSUE TITLE
Experiment with DBSCAN on a suitable dataset

### ISSUE THREAD

**Issue by balapriyac** [sanscript-tech/hacking-tools-scripts#244]
-  When are density based clustering approaches more relevant?
- Give a practical use case of DBSCAN

**Comment by Tejas1510** [comment]
Can i work on this?
I was having a doubt that we need to give only answer to the above question in the readme file

**Comment by balapriyac** [comment]
Hi
You should also make sure to answer these questions as well

**Comment by sravani-7701** [comment]
Hey I would like to work on this issue. But please tell me just we have to describe about DBSCAN right ?

**Comment by balapriyac** [comment]
Yes,identify correct use cases as to when you'd need such clustering
approach

Also, pls be sure to choose such a dataset

**Comment by balapriyac** [comment]
@sravani-7701 are you interested in trying?
